day_6/exercises/my_car.rb

```diff
@@ -0,0 +1,39 @@
+class MyCar
+  attr_reader :year, :color, :model
+  def initialize(year, color, model, speed=0, car_off=true, car_break=0)
+    @year = year
+    @color = color
+    @model = model
+    @speed = speed
+    @car_off = car_off
+    @car_break = car_break
+
+  end
+  def speed_up(number)
+    @speed += number
+    puts "Let's go faster! Current speed: #{@speed} mph"
+  end
+  def car_descelerate(number)
+    @speed -= number
+  end
+  def car_on
+    true
```

Comment:
What is true here?

Reply:
What attribute should you change?

Reply:
I see. I should have set it up as modifying @car_off which is by default true.
def car_on
  @car_off = false
end

Is that what in the ballpark of what you were looking for? If not, can you please explain a bit farther? 
thank you for taking the time to review my exercises!